src/aztec/interfaces/IRollupProcessor.sol

```diff
@@ -111,4 +113,21 @@ interface IRollupProcessor {
     function getSupportedAssets() external view returns (address[] memory, uint256[] memory);
 
     function getSupportedBridges() external view returns (address[] memory, uint256[] memory);
+
+    function convert(
```

Comment:
I did this temporarily because I was thinking that convert was called on rollupProcessor still and my fxn call wasn't working because it wasn't defined in interface. Realized after the change was on purpose and used sendDefiRollup instead. I have fixed this.